broker/lib/controllers/ServiceBrokerApiController.js

```diff
@@ -7,6 +7,7 @@ const errors = require('../errors');
 const utils = require('../utils');
 const catalog = require('../models/catalog');
 const FabrikBaseController = require('./FabrikBaseController');
+const lockManager = require('../../../eventmesh').lockManager;
```

Comment:
Can we stick to a standard on naming convention. This is starting with lower case, all other in upper case.

Reply:
Figured that lockManager is an Instance of ApiServerLockManager class. hence kept it camelcase. Will change if you feel otherwise.

Reply:
Better rename lockManager ==> LockManager. Keep as lower priority as functionaly/modularization changes should take more priority.

Reply:
@sauravmndl Amit is right. Class References exposed are named the way you suggested but instance are proper camel case. Look at existing code in index.js of any file.